HISTORY.rst

```diff
@@ -16,6 +16,7 @@ History
 * Updated documentation of anonymizer attributes - ``is_anonymous_vpn`` and
   ``is_hosting_provider`` - to be more descriptive.
 * Added support for ``user_count`` trait for the GeoIP2 Precision webservice.
+* Added ``static_ip_score`` property to ``geoip2.record.Traits`` for GeoIP2 Precision Insights.
```

Comment:
Sorry, I thought I commented on this but it somehow didn't get submitted last time apparently:

Could we call it an attribute rather than a property? I believe that is the correct term for Python.

Also could we add a "the" before the name? The user_count line has this issue too.

And finally, maybe we could add something about the meaning of the output? We do this for some of the other API libraries as well. Something like https://github.com/maxmind/GeoIP2-php/pull/142/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR21.